### DISCUSSION TITLE
GitHub Actions always failed when I create pr.

### DISCUSSION THREAD

**Discussion by jxnu-liguobin** [kobylynskyi/graphql-java-codegen#561] [General]
@kobylynskyi Can close sonor for PR submission? Maybe style checking and some plugins can implemented it.
 It's too bad that I always get failed. 🤕

**Comment by kobylynskyi** [comment]
Unfortunately, the Sonar token is not propagated to forked repositories. So all PRs that are created from forked repos to the base repo will fail :(

I am open to any ideas on how to fix that.

**Reply by jxnu-liguobin** [reply]
Maybe I should open an issue for sonar's GitHub action? 😿
